src/config.rs

```diff
@@ -168,6 +167,69 @@ pub struct SimpleVocabConfig {
     pub discard_threshold: f32,
 }
 
+/// Options for sizing the vocabulary.
+#[derive(Copy, Clone, Debug)]
+pub enum VocabCutoff {
+    /// Minimum token count
+    ///
+    /// Discard tokens that occur less than the given count.
+    MinCount(MinCount),
```

Comment:
Is it necessary to have these separate struct as opposed to an enum of `MinCount(usize)` and `TargetVocabSize(usize)`?